engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/inputSettings/InputSettingsScreen.java

```diff
@@ -307,9 +317,18 @@ public void onClosed() {
         bindsManager.registerBinds();
 
         // TODO: Find a better place to do this in.
-        TabbingManager.tabForwardInput = bindsManager.getBindsConfig().getBinds(new SimpleUri("engine:tabbingUI")).get(0);
-        TabbingManager.tabBackInputModifier = bindsManager.getBindsConfig().getBinds(new SimpleUri("engine:tabbingModifier")).get(0);
-        TabbingManager.activateInput = bindsManager.getBindsConfig().getBinds(new SimpleUri("engine:activate")).get(0);
+        BindsConfig bindsConf = bindsManager.getBindsConfig();
+        if (bindsConf != null) {
+            bindsConf.getBinds(new SimpleUri("engine:tabbingUI")).stream().findFirst().ifPresent(input -> {
+                TabbingManager.tabForwardInput = input;
+            });
+            bindsConf.getBinds(new SimpleUri("engine:tabbingModifier")).stream().findFirst().ifPresent(input -> {
+                TabbingManager.tabBackInputModifier = input;
+            });
+            bindsConf.getBinds(new SimpleUri("engine:activate")).stream().findFirst().ifPresent(input -> {
+                TabbingManager.activateInput = input;
+            });
+        }
```

Comment:
That's the actual fix, the rest is simply formatting.
An alternative here would be to return null. I chose to do nothing instead as there might've been a valid configuration before.

Reply:
We talked about this offline - it's not clear to neither of us what the semantic operation here is. Is it "reset" to default values? What should happen if there is no value in the binding config?

As @jdrueckert said this will do nothing in case the binding list is empty. An alternative would be to explicitly set the input to `null`. I don't know who can make this call - @BenjaminAmos ?

```java
bindsConf.getBinds(new SimpleUri("engine:tabbingUI")).stream().findFirst().orElse(null)
```